### ISSUE TITLE
Checklist for Onboarding Ephraim (cloud.gov Pages Support)

### ISSUE THREAD

**Issue by apburnes** [cloud-gov/product#1717]
# Onboarding Checklist 

In order to get Ephraim productively contributing to the cloud.gov team, Brian should help Ephraim complete a prescribed set of tasks that will bring them up to speed and get them setup with cloud.gov.

### Required items for all team members

These items help us fulfill security and compliance requirements (including for FedRAMP). If you get stuck, or if these requirements are confusing, ask for help from your buddy or in a cloud.gov channel.

- [x] Take judicious notes on what about this onboarding process or cloud.gov is confusing or frustrating. If you notice a problem (especially with things like documentation), you are more than welcome to fix it! At the very least, please share this information with your onboarding buddy (or someone) at some point so we can make the team/platform better. (You can also file issues and pull requests on [the template Onboarding checklist](https://github.com/cloud-gov/product/blob/master/OnboardingChecklist.md).
- [x] Be sure to introduce yourself and follow up with your onboarding buddy (they should have reached out to you at this point; if they haven't, please let the team know) and make sure this issue is assigned to them in our [Github Project Planning Board](https://github.com/orgs/cloud-gov/projects/2). We use this board to organize, prioritize, and track our work.

#### Pre-requisites

- [x] Complete [GSA OLU](https://gsaolu.gsa.gov/) GSA Mandatory Cyber Security and Privacy Training, including accepting the GSA IT Rules of Behavior, which is required before we can give you access to any cloud.gov systems. If you joined GSA more than two months ago, you've already completed this task and can just check the box.

#### Fulfill security and compliance requirements (including for FedRAMP) - Completed by onboarding buddy

- [x] Make sure they're in [the list of people working on the project](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0).
- [x] Add their name, whether they're Cloud Ops (Platform), and the date they joined the team to the [training tracker](https://docs.google.com/spreadsheets/d/1hqU6cNeEB293OT0j3OvbdAFRkrf2zDOrPVxGfnr4sSw/edit#gid=0). Copy the formulas for the due dates from an existing row (grab the "corner" of the cells and pull down).
- [ ] As they complete training, fill out their completion dates in the [training tracker](https://docs.google.com/spreadsheets/d/1hqU6cNeEB293OT0j3OvbdAFRkrf2zDOrPVxGfnr4sSw/edit#gid=0).
- [x] Review the recurring cloud.gov meetings that are relevant for them in [the team calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_0samf7guodi7o2jhdp0ec99aks@group.calendar.google.com&amp;ctz=America/Los_Angeles) (they will get access to this when added to the cloud.gov Team Google Group).
- [x] Add them to the [`cloud-gov`](https://github.com/orgs/cloud-gov/people) organization in GitHub, and the [`cloud-gov-team`](https://github.com/orgs/cloud-gov/teams/cloud-gov-team) team.

#### Learn our policies and procedures

For the three trainings list at the top, your onboarding buddy will create a separate ticket to track the trainings once scheduling has been finished.  This will help consolidate trainings for multiple new members to the team and prevent them from blocking progress on this onboarding ticket.  Once the trainings are scheduled, they can be marked as complete here.

* [x] Coordinate with your onboarding buddy to go through Contingency Planning training within 60 days (and annually after that). This will cover the following document, which you should also review before or after training:
  * [x] Read the [Contingency Plan](https://docs.cloud.gov/ops/contingency-plan/).
* [x] Coordinate with your onboarding buddy to go through [Incident Response Training](https://docs.google.com/presentation/d/1AZjQE8zBzMRWZIFUuJPkJLted1ykGtALrLPoPRx5Vls/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following document, which you should also review before or after training:
  * [x] Read the [Incident Response Guide](https://cloud.gov/docs/ops/security-ir/).
* [x] Coordinate with your onboarding buddy to go through [nonpublic information training](https://docs.google.com/presentation/d/1uB4MlGCu8ZYUxjKVZKwicQ95MvLxaT4Mh93y6w79GPw/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following documents, which you should also review before or after training:
  * [x] Review the [cloud.gov open source policy guidance about protecting sensitive information](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
  * [x] Read our [sharing secret keys](https://cloud.gov/docs/ops/secrets/#sharing-secret-keys) policy.
  * [x] Review the [TTS requirements for password management](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/password-requirements/).
* [x] Read the [Continuous Monitoring Strategy](https://cloud.gov/docs/ops/continuous-monitoring/), particularly the [cloud.gov team responsibilities](https://cloud.gov/docs/ops/continuous-monitoring/#cloud-gov-team).
* [x] Read the [Configuration Management Plan](https://cloud.gov/docs/ops/configuration-management/).
* [x] Read the [cloud.gov Security Policies and Procedures](https://github.com/cloud-gov/cg-compliance-docs). These documents explain the high-level policies and procedures we must comply with while running cloud.gov, sorted into security control "families" They explain that we follow GSA IT security policy, and they provide a summary of the procedures in our System Security Plan.
* [x] Review the System Security Plan (the latest version lives on [Google Drive](https://drive.google.com/drive/u/0/folders/0B6fPl5s12igNX3JwR2xFZVpmek0); look for "cloud.gov System Security Plan (SSP)" as a *.docx* file). Of particular note for onboarding: Section 9 (System Description) and Section 10 (System Environment)

### Getting to know Federalist
https://docs.google.com/document/d/1tS-9hLb2yHtp500N3obmu0X70XpIMATuPlov2_vDVS0/edit

#### Acceptance Criteria
- [x] Share README
- [x] Add to federalist-admins in federalist-users org.
- [x] Add to daily standup, retro, and sprint planning
- [x] Ensure user has Federalist Github project and brief on its use
- [x] Add to federalist-support, federalist-alerts, and federalist-inquiries Google Groups as needed
- [x] Add them as an agent to https://federalist-support.zendesk.com/agent
- [ ] Add to statuspage.io account
- [x] Add to cloud.gov calendar

### Slack Stuffs

- [x]  Add to @federalist-operators Slack user group

Your onboarding buddy will add you to these Slack channels:

- [x] `#cloud-gov` - bots post announcements here
- [x] `#cg-offtopic` - off-topic team sharing
- [x] `#cg-general` - program-level information and discusion
- [x] `#cg-support` - support requests and assistance within TTS
- [x] `#cg-incidents` - private channel for incident response
- [x] `#cg-priv-all` - private channel for in-team discussion
- [x] `#federalist` - general federalist channel
- [x] `#federalist-support` - general support channel
- [x] `#federalist-logs` - platform deployment messages
- [x] `#federalist-private` - internal team chat
- [x] `#federalist-public` - General public channel
- [x] `#federalist-supportstream` - zendesk updates
